src/pkg/output/format.go

```diff
@@ -35,7 +35,7 @@ func AsTable[T any](dataList []T, columns []string, getColumnData func(T) []map[
 		for _, columnData := range getColumnData(item) {
 			row := tab.Row()
 			for _, column := range columns {
-				value, _ := columnData[column]
+				value := columnData[column]
```

Comment:
src/pkg/output/format.go:38:5: S1005: unnecessary assignment to the blank identifier (gosimple)